sysutils/ntpstat/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        darkhelmet ntpstat 86e0a68df62e7f6b6c5ee97ee9d15b5e28a4b359
+
+name                ntpstat
+version             0.2
+categories          sysutils net
+platforms           darwin
+license             GPL-2
+maintainers         {hotmail.com:amtor @RobK88} \
+                    openmaintainer
+
+description         ${name} shows the network time protocol (ntp) status
+
+long_description    ${name} shows the network time protocol (ntp) status.\
```

Comment:
this first line can/should be replaced by `long_description    ${description}` as not to duplicate things